### DISCUSSION TITLE
checkbox group list

### DISCUSSION THREAD

**Discussion by onno-timmerman** [django-commons/django-unicorn#461] [Q&A]
Could you give an example how you doe checkbox group.

```
{% for name in names %}
<input type="checkbox" name="group" u:model="names" value="{{ name.id }}>{{name}}
<input type="checkbox" name="group" u:model="names" value="{{ name.id }}>{{name}}
{%endfor %}
```

The only solution I can find now is not to use model but click and write some logic to manage the checkbox grouping. In the docs I don't find anything obvious

```
def set_names(self, val): 
   if val in names:
      names.remove(val)
   else:
      names.append(val)
   self.names = list(set(names))
```

**Comment by adamghill** [comment]
Looks like I only have an example for checkbox (but not a group): https://github.com/adamghill/django-unicorn/blob/11d2ea8980fd836088102fbdf7e9760ecf80ab10/example/unicorn/templates/unicorn/html-inputs.html#L4. Would you like me to create an issue to look into this?
